HackingCommon/IDirect3DUnknown.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+#include <Unknwn.h>
+
+class IDirect3DUnknown : public IUnknown
+{
+public:
+	IUnknown* m_pUnk;
+	ULONG m_ulRef;
+	
+	IDirect3DUnknown(IUnknown* pDevice) :
+		m_pUnk(pDevice)
+	{}
+
+	virtual STDMETHODIMP QueryInterface(THIS_ REFIID riid, void** ppvObj);
+};
```

Comment:
IDirect3DUnknown is from Chiri's original code ( https://github.com/Nintynuts/3Dmigoto/commit/1ba6f7e8c8bf6bca3ef49406e7e48698c8d7f4dd#diff-95006bfff5a5f88ff0f40f730d65afaaR5 ), and Bo3b long since got rid of this while re-architecting 3DMigoto from the ground up, so I don't think we want to bring this back.

If you got any references to this it will be in bit-rotted projects that hadn't been re-architected yet and you shouldn't be building if using the "Zip Release" configuration.